erpnext_documentation/www/docs/user/manual/en/setting-up/workflows.md

```diff
@@ -51,8 +51,6 @@ Before creating a Workflow, it is advised to create these first:
 
 ### 2.2 Things to note when creating a Workflow
 
-* Make sure the **Allow Workflow** checkbox is selected in the Form Settings section of the DocType you want use to create a workflow. If it is unchecked, a workflow can not be created for that DocType.
```

Comment:
as discussed with @abhishekbalam the feature wasn't merged, hence reverting docs